src/actions/field-actions.js

```diff
@@ -14,304 +14,316 @@ import isValid from '../form/is-valid';
 import NULL_ACTION from '../constants/null-action';
```

Comment:
Diff again makes it look like I changed more than I did.  The notable changes were:

1. Added the defaultStrategy
2. Moved bulk of content into the createFieldActions function
3. Converted all usage of _get() to the strategy s.get()
4. Moved the trackable() wrapping to the return statement using mapValues()